### PR TITLE
Add Dockerfile and support files

### PR DESCRIPTION
**Pull request by mbklein**
Changes in this pull request:

- Adds Dockerfile to allow automated docker builds
- Adds support files (under `./docker`) to make running under Docker quick & easy

This should make trying out LAKEsuperior much quicker & easier for anyone with a docker environment (or someone who's willing to tinker). It will also allow for the creation of an automated build on Docker Hub for anyone who just wants to pull & go.

@scossu

### CODE REVIEW

**Comment by scossu** [comment]
Thanks @mbklein , this is a great enhancement. I have limited familiarity with Docker so I won't be able to review this thoroughly, but I'll add some comments for what I think I know...

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM        python:3.5
+MAINTAINER  Michael B. Klein <michael.klein@northwestern.edu>
+RUN         mkdir -p /var/app /data
+WORKDIR     /var/app
```
Comment (scossu) [inline Dockerfile]: I believe the UNIX standard is to use `/var` for data files and `usr` for application data. Unless you are familiar with different setups, shall this be set to `usr/local`?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+ADD         . lakesuperior
+WORKDIR     /var/app/lakesuperior
```
Comment (scossu) [inline Dockerfile]: See comment to line 4.

**Comment by mbklein** [comment]
Yes, I’ll be happy to. There’s a bit of a chicken and egg situation due to the fact that running on Docker will be easier if you set up a webhook to have Docker Hub do automated builds. But the Dockerfile has to be present before that happens. I’ll add local instructions and then update them to integrate Docker Hub later.

File: Dockerfile (hunk 2 above)
Comment (mbklein) [reply Dockerfile]: I’m happy to put this anywhere. I was following the AWS and (sometime) Docker convention of using /var/app, but I agree that /usr/local makes sense. I’ll change it.

**Pull request merged**